pkg/cri/server/container_stats_list.go

```diff
@@ -61,11 +64,85 @@ func (c *criService) toCRIContainerStats(
 		if err != nil { //nolint:staticcheck // Ignore SA4023 as some platforms always return nil (metrics unimplemented)
 			return nil, fmt.Errorf("failed to decode container metrics for %q: %w", cntr.ID, err)
 		}
+
+		// this is a calculated value and should be computed for all OSes
+		nanoUsage, err := c.getUsageNanoCores(cntr.Metadata.ID, false, cs.Cpu.UsageCoreNanoSeconds.Value, time.Unix(0, cs.Cpu.Timestamp))
```

Comment:
not sure why the code was refactored, always passing false even for the sandbox "pause" container?

Reply:
kk code works, fix is fine.. later we can refactor this so windows/ other impls of containerMetrics() have usage nano as well.. and the other stats necessary

Reply:
I guess could have cleaned that up a bit more.  merged much faster than I expected :-).  

I can revisit this in https://github.com/containerd/containerd/pull/7099 once we have some agreement on  WindowsPodsandbox stats for https://github.com/kubernetes/enhancements/pull/3439